pkg/converter/testdata/docker-compose-v2.yml

```diff
@@ -10,8 +10,7 @@ services:
     links:
       - db
   wordpress:
-   depends_on:
-     - db
```

Comment:
I don't quite understand that part.

Since you mention host:port search: Env vars could reference other services without specifying a port. Is that covered anywhere?